src/components/App.js

```diff
@@ -18,17 +20,23 @@ export const App = ({ i18n, dir }) => {
   }, [dirState])
   // end of useDirection
 
+  useEffect(() => {
+    document.body.className = fontSizeClass
+  }, [fontSizeClass])
+
```

Comment:
```suggestion
  // useFontSize
  const [fontSizeClass, setFontSizeClass] = useState(articleTextSize.getFontSizeClassName())
  useEffect(() => {
    document.body.className = fontSizeClass
  }, [fontSizeClass])
  // useFontSize
```

can we do this scope same as `useDirection`, so we know this scope belongs to text size related code

Reply:
Sure thing